packages/find-replace/src/getSearchHighlightDecorate.ts

```diff
@@ -0,0 +1,33 @@
+import {Decorate, getPlatePluginOptions} from '@udecode/plate-core'
+import {NodeEntry, Range, Text} from 'slate'
+import {MARK_SEARCH_HIGHLIGHT} from './defaults'
+import {DecorateSearchHighlightOptions} from './types'
+
+export const getSearchHighlightDecorate =
+  ({search}: DecorateSearchHighlightOptions): Decorate =>
+  editor => {
+    const options = getPlatePluginOptions(editor, MARK_SEARCH_HIGHLIGHT)
+
+    return ([node, path]: NodeEntry) => {
+      const ranges: Range[] = []
+
+      if (search && Text.isText(node)) {
+        const {text} = node
+        const parts = text.toLocaleLowerCase().split(search.toLocaleLowerCase())
```

Comment:
Here we converted both editor value and search string to be lowercase, so it's not case sensitive anymore.